tensorflow/python/keras/metrics.py

```diff
@@ -3342,6 +3342,9 @@ def categorical_accuracy(y_true, y_pred):
   Returns:
     Categorical accuracy values.
   """
+  #assert if both have same shape
```

Comment:
Nit: space between comment mark and text, and prefer a more specific comment:

\# assert if predicted and true labels have the same shape

Reply:
Noted.